roles/firewall/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+
+- name: Install 'ufw'
+  pacman: name=ufw state=present
+
+- name: Install 'ufw-extras'
+  pacman: name=ufw-extras state=present
+
+- name: Allow ssh connections
+  ufw:
+    rule: allow
+    port: ssh
+    proto: tcp
+
+- name: add sysctl configuration
+  template:
+    dest: /etc/ufw/sysctl.conf
+    src: sysctl.conf.j2
+
+- name: Enable UFW
+  ufw:
+    state: enabled
+
+- name: Set deny default policy
+  ufw:
+    policy: deny
+
+- name: Set allow default policy for forwarding
```

Comment:
Why are we editing the sample config? Shouldn't we be installing our own config to `/etc/ufw/ufw.conf`?

Reply:
I added a template to build our own config, can you review?